.ci/ansible/roles/common/tasks/setup_test_script.yml

```diff
@@ -0,0 +1,38 @@
+---
+- name: Extend environment
+  lineinfile:
+    state: present
+    line: "{{item}}"
+    insertafter: EOF
+    dest: "/home/{{ansible_user}}/e2e-testing/.env"
+    create: yes
+  with_items:
+    - "SUITE={{ lookup('env', 'SUITE') or 'fleet' }}"
+    - "STACK_VERSION={{ lookup('env', 'STACK_VERSION') or '8.0.0-SNAPSHOT' }}"
+    - "BEAT_VERSION={{ lookup('env', 'BEAT_VERSION') or '8.0.0-SNAPSHOT' }}"
+    - "ELASTIC_APM_GLOBAL_LABELS={{ lookup('env', 'ELASTIC_APM_GLOBAL_LABELS') }}"
+
+- name: Create test script file
+  become: no
+  copy:
+    owner: "{{ansible_user}}"
+    group: "{{ansible_user}}"
+    mode: '0777'
+    dest: "/home/{{ansible_user}}/e2e-testing/.ci/scripts/functional-test.sh"
```

Comment:
We already have a script to run the tests in the same path. Do we want to use the old one or we don't care about overriding it?

Reply:
I trimmed this one down for VM runs only, at least until I get green across the board. It's mainly to help with debugging atm